modules/layers/src/index.ts

```diff
@@ -37,3 +37,6 @@ export {default as SolidPolygonLayer} from './solid-polygon-layer/solid-polygon-
 // Experimental layer exports
 export {default as _MultiIconLayer} from './text-layer/multi-icon-layer/multi-icon-layer';
 export {default as _TextBackgroundLayer} from './text-layer/text-background-layer/text-background-layer';
+
+// Font Atlas Manager
+export {setFontAtlasCacheLimit} from './text-layer/font-atlas-manager';
```

Comment:
Regarding this API - we don't have other setters like this in the library. My personal preference is to expose it as a static member of `TextLayer`, i.e.

```js
TextLayer.fontAtlasCacheLimit = 5;
```

Reply:
As you suggested, I don't expose the function anymore and instead added a new static setter to TextLayer.